index.html

```diff
@@ -17,12 +16,29 @@
                 </div>
                 <!-- Toolbar Menu -->
                 <nav class="toolbar-menu-container">
-                    <img src="images/Icon - Menu.svg" alt="Menu icon" />
-                    <ul class="navbar-links-container">
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
-                    </ul>
+                    <a href="#">
+                        <img src="images/Icon - Menu.svg" alt="Menu icon"
+                            id="menu-icon" />
+                    </a>
+                    <!-- mobile menu -->
+                    <div class="mobile-menu-container mobile-menu-toggler">
+                        <div class="menu-icon-container">
+                            <a><img
+                                    src="./images/mobile_exit_Icon.png"
+                                    alt="exit icon"
+                                    id="exit-menu-icon"
+                                    /></a>
```

Comment:
Kindly consider taking a second look at your `images` folder as there seems to be no `image` with the name `mobile_exit_Icon.png` due to your close image isn't displaying properly on your page.

<img width="336" alt="Screenshot 2023-04-17 at 17 13 50" src="https://user-images.githubusercontent.com/107192076/232560430-8b8b0435-6f33-4602-9935-22e132e9e8c1.png">